src/SFA.DAS.Campaign.UnitTests/Infrastructure/Api/WhenCallingGet.cs

```diff
@@ -21,7 +23,8 @@ public class WhenCallingGet
         [Test, AutoData]
         public async Task Then_The_Endpoint_Is_Called_With_Authentication_Header_And_Data_Returned(
             List<string> testObject, 
-            OuterApiConfiguration config)
+            OuterApiConfiguration config,
+            Mock<IHtmlControlAbstractFactory> controlFactory)
```

Comment:
Based on earlier comment might need to undo this part passing in the`IHtmlControlAbstractFactory`